templates/news-list/news-list.css

```diff
@@ -0,0 +1,61 @@
+main > div {
+    
+    
+    /* left nav */
+    aside {
+        width: auto;
+        box-sizing: border-box;
+        z-index: 1;
+        position: relative;
+        float: left;
+        
+    }
+
+     /* right content */
+    section {
+        position: relative;
+        flex: 1;
+        width: 95%;
+        background-color: #fff;
+        box-sizing: border-box;
+        padding-left: 35%;
+        
+        
+
+        /* border */
+        &::before {
+            content: '';
+            position: absolute;
+            top: 0;
+            left: 35%;
+            width: 1px;
+            height: 100%;
+            background: #ccc;
+            box-sizing: border-box;
+        }
+    }
+
+    .articles {
+        width: auto;
+       
+        
+    }
+
+    .card {
+        width: auto;
+    }
+
+    .card h3 {
```

Comment:
awesome nesting! note you can next the h3 inside of the card selector, that's the power of nesting, you only write the selector once, then all the nested elements go inside.  Helps keep things clean and segmented :)